docs/source/docs_guide.md

```diff
@@ -181,8 +263,34 @@ instructions show you how to use ReadTheDocs to do this.
    `Branch or tag deletion`.
 10. Click `Add webhook`.
 
-Now anytime you modify or add documentation content to your fork, this
-URL will automatically get updated with your changes!
+Use `fabric-docs-i18n` instead of `fabric` in the above instructions if you're
+building an international language translation.
+
+Now, anytime you modify or add documentation content to your fork, this URL will
+automatically get updated with your changes!
+
+## Making a PR
+
+You can submit your PR for inclusion using the following
+[instructions](./github/github.html#opening-a-pull-request-in-github).
```

Comment:
So many people are submitting doc updates without signing their requests. I think we have to drive them down this path instead https://hyperledger-fabric.readthedocs.io/en/latest/github/github.html
where it shows them to do the commit -s